Atomics load add

rwalczyna opened the pull request:
Parallel tests:
Before: +1475 | -1005
After : +1516 | -964


#63 Should be merged first.

rerobika commented:
Please rebase.

rerobika reviewed (changes requested):
Only one request. Can you separate these operations into an individual source file?

rerobika commented:
Also please keep in mind during rebasing that 3d43aa82e826ba6cfb787f412f88de55a6b30182 moved the `deps/jerry/jerryscript` directory into a submodule.

rwalczyna commented:
I rebased the commit and moved Atomics methods to jerry_atomics.
I also moved functions used to register handlers (jerryx_handler_register_*) to utils - to use them outside of jerry_isolate.

'parallel' suite before (3d43aa82e826ba6cfb787f412f88de55a6b30182):
[14:30|% 100|+ 1588|- 892]: Done
after:
[12:38|% 100|+ 1640|- 840]: Done

rerobika commented on deps/jerry/v8jerry/v8jerry_atomics.cpp:
```diff
@@ -0,0 +1,122 @@
+    return;
+}
+
+}  // namespace JerryAtomics
```
Please add new line at the end of the file.

rerobika commented on deps/jerry/v8jerry/v8jerry_utils.cpp:
```diff
@@ -150,3 +150,49 @@ void JerryForceCleanup(void) {
+
+    JerryxHandlerRegister (name_p, prototype_val, handler_p);
+    jerry_release_value (prototype_val);
+}
```
Ditto.

rerobika reviewed (commented):
LGTM, after these minor changes.

rerobika reviewed (approved):
LGTM
